### PR TITLE
feat: ckBTC transactions on wallet detail page

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Add CTA to execute a ckBTC transaction on wallet detail page.

# Changes

- add footer action to instantiate ckBTC transaction and pass selected account
- modify dispatch on success to reload account data
- duplicate properties fee and token as [requested](https://github.com/dfinity/nns-dapp/pull/1812#discussion_r1095949160)

# Screenshot

demo2

### CODE REVIEW

File: frontend/src/lib/modals/accounts/CkBTCTransactionModal.svelte [hunk 1]
```diff
@@ -46,31 +46,26 @@
 
     if (success) {
       toastsSuccess({ labelKey: "accounts.transaction_success" });
-      dispatcher("nnsClose");
+      dispatcher("nnsTransfer");
```
Comment (lmuntaner) [inline frontend/src/lib/modals/accounts/CkBTCTransactionModal.svelte:49]: Why did you change this event? Should we change it in all modals?

AFAIK, the modal takes care of the success and error messages. Therefore, there was no need to expose that outside.

**Review by lmuntaner — APPROVED** [review]
Just a minor question, but already approving. LGTM!

File: frontend/src/lib/modals/accounts/CkBTCTransactionModal.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/modals/accounts/CkBTCTransactionModal.svelte:49]: > modify dispatch on success to reload account data

To reload the account after transfer because otherwise the new balance is not reflected in screen.

In Nns we syncAccounts and therefore it works out.

In Sns wallet it currently works out because for some reason the `accountIdentifier` is refreshed after transfer and per extension the account is reloaded. Might need further inspection here but, it works out so fine so far.

Comment (lmuntaner) [reply frontend/src/lib/modals/accounts/CkBTCTransactionModal.svelte:49]: ok, thanks for the explanation
